web/public/mgmt/config.yml

```diff
@@ -658,13 +658,85 @@ collections:
                             widget: object
                             fields:
                               - { label: "Description", name: "description", widget: "markdown" }
+                  - label: Trustees
+                    name: trustees
+                    collapsed: true
+                    widget: object
+                    fields:
+                      - { label: "Label", name: "label", widget: "string" }
+                      - { label: "Add Button Text", name: "addButtonText", widget: "string" }
+                      - { label: "Error", name: "error", widget: "string" }
+                      - { label: "Success snackbar - body", name: "successSnackbarBody", widget: "string" }
+                      - { label: "Modal Title", name: "modalTitle", widget: "string" }
+                      - { label: "Subheader", name: "subheader", widget: "string" }
+                      - { label: "Trustee Full Name", name: "trusteeFullName", widget: "string" }
+                      - { label: "Placeholder", name: "placeholder", widget: "string" }
+                      - {
+                          label: "Success snackbar - header",
+                          name: "successSnackbarHeader",
+                          widget: "string",
+                        }
+                      - { label: "Modal save button", name: "modalSaveButton", widget: "string" }
+                      - { label: "Description", name: "description", widget: "string" }
                   - label: Withdrawals
                     name: withdrawals
                     collapsed: true
                     widget: object
                     fields:
                       - { label: "Label", name: "label", widget: "string" }
                       - { label: "Body", name: "body", widget: "markdown" }
+                  - label: Veteran Nonprofit
+                    name: veteranNonprofit
+                    collapsed: true
+                    widget: object
+                    fields:
+                      - { label: "Label", name: "label", widget: "string" }
+                      - { label: "Description", name: "description", widget: "string" }
+                      - { label: "Radio no text", name: "radioNoText", widget: "string" }
+                      - { label: "Radio yes text", name: "radioYesText", widget: "string" }
+                      - { label: "Error", name: "error", widget: "string" }
+                      - {
+                          label: "Domestic nonprofit veteran corp review text",
+                          name: "domNpVetCorpReviewText",
+                          widget: "string",
+                        }
+                      - {
+                          label: "Board members yes review text",
+                          name: "yesBoardMembersReviewText",
+                          widget: "string",
```

Comment:
[pebble] No just this item, but ANY config that contains markdown syntax like the `**` bolding needs to be widget type `markdown`